src/reporter.js

```diff
@@ -94,7 +150,8 @@ module.exports = class TestrunnerReporter {
         /**
          * only upload assets if a session was initiated before
          */
-        if (!sessionId) {
+
+        if (!sessionId || 0 === sessionId) {
```

Comment:
`0 === sessionId` is redundant. If it's `0`, then `!sessionId` will evaluate to `true`.

Reply:
updated. Thanks!